static/js/src/modules/feedback.js

```diff
@@ -0,0 +1,66 @@
+const { el, mount, textContent, list, setChildren, setStyle, setAttr } = redom
+
+let pageTitle = document.querySelector(".blog-post-title")
+
+let title = `Potential Missing Documentation: ${pageTitle.textContent}`
+let body = `
+Describe the missing content:
+
+Where would you have expected to find the information:
+
+`
+
+const newIssueQuery = `title=${encodeURIComponent(title)}&body=${encodeURIComponent(body)}`
+
+class FeedBack {
+    constructor() {
+        this.prompt = el("div.statement", "find what you are looking for?")
+
+        this.thumbsUp = el("button.feedback-button", {
+            onclick: function (e) {
+                this.closeFeedback()
+            }.bind(this)
+        }, "👍")
+        this.thumsDown = el("button.feedback-button", {
+            onclick: function (e) {
+                this.notFindingFeedback()
+            }.bind(this)
+        }, "👎")
+
+        this.buttonContainer = el("div.feedback-response-container", this.thumbsUp, this.thumsDown)
+
+        this.actionLink = el("a", { href: `https://github.com/fermyon/developer/issues/new?${newIssueQuery}`, target: "_blank" }, "open Issue")
+        this.el = el("div.feedback-modal", this.prompt, this.buttonContainer)
+        let contentArea = document.querySelector("main")
+        console.log(contentArea)
+        let a  = setInterval(() => {
+            let percentage = window.scrollY / (document.body.offsetHeight - window.innerHeight) ;
+            if (percentage > 0.5) {
+                setStyle(this.el, { opacity: 1, "z-index": 999 })
+                clearInterval(a)
+            }
+        }, 1000);
+
+    }
+
+    closeFeedback() {
+        setStyle(this.el, { opacity: 0 })
+        setTimeout(() => {
+            this.prompt.textContent = "Still finding what you need?"
+            setStyle(this.el, { opacity: 1, "z-index": 999 })
+        }, 5000);
+    }
+
+    notFindingFeedback() {
+        this.prompt.textContent = "Can you help us improve?"
```

Comment:
Rather than ask a favour (help us improve), let's offer them help.

Top message can read "Let us help you!"
The button can read "Open a GitHub Issue."

This way they known that are opening GitHub UI (not a surprise).

Reply:
@mikkelhegn I am not sure the question fits well in the context because we are not allowing text input but instead redirecting them to GIthub issues which might be a better place to ask the question.